src/librbd/mirror/snapshot/GetGroupInfoRequest.h

```diff
@@ -0,0 +1,92 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+
+#ifndef CEPH_LIBRBD_MIRROR_SNAPSHOT_GET_GROUP_INFO_REQUEST_H
```

Comment:
How about renaming this to GroupGetInfoRequest ? Starting all group class names with "Group" will make listing any the Group related classes (which will be written in the future) easier. Or else move them into a group namespace , which might be overkill.

Reply:
@nbalacha I've renamed as GroupGetInfoRequest. Also renamed CreatePrimaryGroupRequest to GroupCreatePrimaryRequest, and renamed PreareGroupImages to GroupPrepareImages.